addons/mail/static/src/components/message/message.xml

```diff
@@ -138,6 +140,9 @@
                                             'o-message-selected': props.isSelected,
                                             'o-mobile': env.messaging.device.isMobile,
                                         }" t-on-click="_onClickReply" title="Reply"
+                                        role="button"
+                                        tabindex="0"
+                                        aria-label="Reply"
```

Comment:
no extra lines (applies to whole diff)

Reply:
I would've preferred separated lines, but done.